### PR TITLE
media-fonts/iosevka: 2.3.3 bump, add style sets USE

### PR DESCRIPTION
**Pull request by 0xd34df00d**
Closes: https://bugs.gentoo.org/696466
Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Georg Rudoy <0xd34df00d@gmail.com>

### CODE REVIEW

File: media-fonts/iosevka/iosevka-2.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit font
+
+DESCRIPTION="Open source coding font"
+HOMEPAGE="http://be5invis.github.io/Iosevka "
```
Comment (charIes17) [inline media-fonts/iosevka/iosevka-2.3.3.ebuild:9]: Use https

**Comment by eternal-sorrow** [comment]
Yes, building from source code is quite possible, but requires some tools, such as ttfautohint and otfcc, that are not in the portage tree, so those need to be added first.  Also, it would be great to add savedconfig option to provide user-written TOML config for custom build.

**Comment by juippis** [comment]
If someone's up to the task, I don't mind adding those packages to the tree simultaneously with a reworked package. 

I think it's safe to close this, iosevka-4.5.0 is already in tree. I'll open a bug to suggest and track the building from source possibility.

**Comment by juippis** [comment]
https://bugs.gentoo.org/769263

**Pull request closed without merging**